crates/topos-p2p/src/runtime/handle_event/grpc.rs

```diff
@@ -0,0 +1,19 @@
+use bytes::BufMut;
+use bytes::BytesMut;
+use http::HeaderMap;
+use http_body::Body;
+use http_body_util::BodyExt;
+use tokio::io::AsyncWriteExt;
+use tonic::client::GrpcService;
+use tracing::info;
+
+use crate::{behaviour::grpc, Runtime};
+
+use super::EventHandler;
+
+#[async_trait::async_trait]
+impl EventHandler<grpc::Event> for Runtime {
+    async fn handle(&mut self, event: grpc::Event) {
+        info!("gRPC event: {:?}", event);
```

Comment:
Hmm, this isn't doing much! What do we need this for? Is it a placeholder for other code?